akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowGraphCompileSpec.scala

```diff
@@ -440,6 +440,32 @@ class FlowGraphCompileSpec extends AkkaSpec {
         }
       }
 
+      "Junction is connected through GraphFlow" in {
+        val gflow = Flow[Int, String]() { implicit builder ⇒
+          import FlowGraphImplicits._
+
+          val in = UndefinedSource[Int]
+          val out = UndefinedSink[String]
+
+          in ~> Flow[Int].map(_.toString) ~> out
+
+          (in, out)
```

Comment:
looking at this pair feels confusing every time. Maybe we should have a case class instead of a raw pair? At least when using a case class the parameters can be named.